cli/cmd/tapRunner.go

```diff
@@ -65,24 +66,25 @@ func RunMizuTap() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel() // cancel will be called when this function exits
 
-	targetNamespace := getNamespace(kubernetesProvider)
+	targetNamespaces := getNamespaces(kubernetesProvider)
+
 	var namespacesStr string
-	if targetNamespace != mizu.K8sAllNamespaces {
-		namespacesStr = fmt.Sprintf("namespace \"%s\"", targetNamespace)
+	if !mizu.Contains(targetNamespaces, mizu.K8sAllNamespaces) {
```

Comment:
you don't really need to check here "contains" you just need that the targetNamespaces[0] == mizu.K8sAllNamespaces